common/avroSchema_v1.json

```diff
@@ -70,7 +70,8 @@
       },
       "required": [
         "type"
-      ]
+      ],
+      "additionalProperties": false
```

Comment:
I dont think these changes with `additionalProperties` is necessary, and on the other side, are you not allowed to have additional properties in Avro? 🤔

Reply:
As I remember Avro is allows only defined attributes. For example Record - https://avro.apache.org/docs/1.11.1/specification/#schema-record

Reply:
Seems like additional properties is allowed on the different levels of Avro: https://asyncapi.slack.com/archives/C0230UAM6R3/p1712059889567579?thread_ts=1711713980.293369&cid=C0230UAM6R3

Reply:
I removed restriction